lib/routes/authorization.js

```diff
@@ -19,6 +18,13 @@ const verify = require('../browserid');
 const CODE = 'code';
 const TOKEN = 'token';
 
+const UNTRUSTED_CLIENT_SCOPES = [
+  'profile:uid',
+  'profile:email',
+  'profile:avatar',
```

Comment:
In discussion with @ryanfeeley he felt pretty strongly that we should not share the avatar by default.